### PR TITLE
feat: replace LESS with emotion css

### PR DESCRIPTION
**Pull request by przemvs**
Refactor LESS style to Emotion CSS.

### CODE REVIEW

File: src/script/components/ParticipantItemContent/ParticipantItemContent.tsx [hunk 1]
```diff
@@ -101,7 +112,7 @@ export const ParticipantItemContent = ({
         </button>
       )}
 
-      {showArrow && <Icon.ChevronRight className="chevron-right-icon participant-item__content__chevron" />}
```
Comment (przemvs) [inline src/script/components/ParticipantItemContent/ParticipantItemContent.tsx:104]: chevron-right-icon not exist in the code and is not necessary.

File: src/style/content/preferences/account.less [hunk 2]
```diff
@@ -177,7 +177,7 @@
 .preferences-account-username-wrapper {
   position: relative;
   width: 256px;
-  min-height: @line-height-xl * 2;
+  min-height: calc(@line-height-xl * 2);
```
Comment (atomrc) [inline src/style/content/preferences/account.less:180]: why do you need a `calc` here? Doesn't seem to be a dynamic value 🤔

Comment (V-Gira) [reply src/style/content/preferences/account.less:180]: need to use calc with the css variable is my guess

`@line-height-xl` is no longer a rem value but `var(--line-height-xl)`

Comment (atomrc) [reply src/style/content/preferences/account.less:180]: But there is no css variable in there, only `less` variable (which are computed at compile time)

Comment (atomrc) [reply src/style/content/preferences/account.less:180]: ooohhhh the `less` variable contains a `css`  variable. 
Humm then we should probably use the css variable here, and avoid having the `less` proxy. WDYT?

File: src/style/components/list/participant-item.less [hunk 3]
```diff
@@ -100,97 +96,6 @@
...
-      &__name-wrapper {
-        display: flex;
-        overflow: hidden;
-        width: 100%;
-        padding-right: 8px;
-        color: var(--main-color);
-      }
-
-      &__name {
-        .ellipsis;
```
Comment (atomrc) [inline src/style/components/list/participant-item.less:160]: Are you absolutely sure those selectors are not used anywhere else?

File: src/style/content/preferences/account.less (hunk 2 above)
Comment (V-Gira) [reply src/style/content/preferences/account.less:180]: yes, this makes sense

File: src/style/components/list/participant-item.less (hunk 3 above)
Comment (V-Gira) [reply src/style/components/list/participant-item.less:160]: after looking at `ParticipantItemContent.tsx` , looks good to me

Comment (atomrc) [reply src/style/components/list/participant-item.less:160]: But that's the thing, you need to look at all the places that are **not** `ParticipantItemContent.tsx` and check that those classes do not exist somewhere else before we can safely remove them from the CSS file. 

That's the main problem with CSS, it's global and never scoped so removing declarations requires some rigorous checks

Comment (V-Gira) [reply src/style/components/list/participant-item.less:160]: You're right, I'm forgetting that you could pass say `.participant-item__content__chevron` outside of  `ParticipantItemContent.tsx` if you wanted to

File: src/style/content/preferences/account.less (hunk 2 above)
Comment (przemvs) [reply src/style/content/preferences/account.less:180]: Sure, we should avoid proxies with started with '@' and replace to css variables :) @atomrc

File: src/style/components/list/participant-item.less (hunk 3 above)
Comment (przemvs) [reply src/style/components/list/participant-item.less:160]: All fine there. :) In another PR the participant-item.less is completely removed. :)

**Review by atomrc — APPROVED** [review]

**Pull request merged**